website/server/controllers/api-v3/chat.js

```diff
@@ -222,7 +222,8 @@ api.flagChat = {
 
     if (!message) throw new NotFound(res.t('messageGroupChatNotFound'));
 
-    if (message.uuid === user._id) throw new NotFound(res.t('messageGroupChatFlagOwnMessage'));
+    // Your own message should be flaggable
+    //if (message.uuid === user._id) throw new NotFound(res.t('messageGroupChatFlagOwnMessage'));
```

Comment:
Please just remove this. Also, I bet the build will fail because there is a test that covers not being able to flag your own message.

The test should be updated to illustrate that you _can_ flag your own messages.

Also, the `messageGroupChatFlagOwnMessage` translation string should be removed.

```bash
git grep "messageGroupChatFlagOwnMessage" common/locales/en
```

^ to find the file it is in